.prettierignore

```diff
@@ -1,5 +1,4 @@
 # Ignore artifacts:
 node_modules
 dist
-TODO.md
```

Comment:
Already Updated in the branch.
no Idea why it's showing like this here ( I searched in the previous commits and reset the code but still have the same problem).